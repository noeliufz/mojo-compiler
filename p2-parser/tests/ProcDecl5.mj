proc p(i, j: int) {}
