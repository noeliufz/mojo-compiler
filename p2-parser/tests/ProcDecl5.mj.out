proc p (
    val i: int;
    val j: int;
  )
{
}
